app/assets/stylesheets/pages/my-notifications.scss

```diff
@@ -37,6 +37,9 @@
             font-weight:$regular;
         }
     }
+    .all-notifications{
+        opacity: .9;
+    }
```

Comment:
Could you remove these three lines please (there's a `.read` selector used for read notifications on line 46/49, so we can reuse those styles)